lesson_10/lesson_10_task_3/Main.java

```diff
@@ -0,0 +1,41 @@
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Person<String> person1 = new Person<>("Charley");
+        Person<String> person2 = new Person<>("Bob");
+        Person<String> person3 = new Person<>("Jack");
+
+        // а) Создать динамический массив (ArrayList), содержащий объекты класса Person.
+        ArrayList<String> list1 = new ArrayList<>();
```

Comment:
содержащий объекты **класса Person.** не **String**. Также лучше использовать интерфейс.
```suggestion
        List<Person> list1 = new ArrayList<>();
```